jsapp/js/account/plans/useDisplayPrice.hook.tsx

```diff
@@ -0,0 +1,26 @@
+import {useMemo} from 'react';
+import {Price} from 'js/account/stripe.types';
+import {getAdjustedQuantityForPrice} from 'js/account/stripe.utils';
+
+export const useDisplayPrice = (
+  price?: Price | null,
+  submissionQuantity = 1
+) => {
+  return useMemo(() => {
```

Comment:
I guess the purpose of making this a hook and not just a simple "selector" function is to embed the useMemo? Just checking my understanding. Does it offer an advantage over a function without it? The memo depends on the passed in submissionQuantity, price. I'm not asking whether useMemo is merited generally or not, but is it actually executing the way I think, where it runs again if the variable passed in changes, in the scope of where the function is called from.

Reply:
Correct - instead of the hook being run on every render, it only updates once when the price and/or submission quantity change. It's a small optimization, but if I add a `console.log` into the body of the `useMemo()`, then test without the `useMemo()`, it does log fewer times.